### DISCUSSION TITLE
VueformBuilder preview$ and el$.form$.schema

### DISCUSSION THREAD

**Discussion by dan-dazlab** [vueform/builder#89] [Questions]
In Vueform I can access schema from any element with `el$.from$.schema`.
But this doen't work in VueformBuilder.
I can access it only from instance: `this.$refs.builder$.builder.schema`

other problem is with 
```js
const loadingPromise = this.$refs.builder$.load(form);
loadingPromise.then(() => {
      // wait for this.$refs.builder$.preview$ to be initialized
      watch(
        () => this.$refs.builder$?.preview$,
        (preview$) => {
          // this is the final loaded position
        },
        { once: true }
      );
    });
```

BTW: when preview$ is loaded this.$refs.builder$.preview$.form$.schema === this.$refs.builder$.builder.schema !== el$.from$.schema

Any suggestions what could go wrong on a custom element?
el$ is const el$ = useElement(props, context)

**Comment by adamberecz** [comment]
You should be able to use `el$.form$.schema` in a custom element, which should return the current form schema even in the builder.

I've created a [sandbox](https://stackblitz.com/edit/github-ya4umq?file=src%2Fcomponents%2FCustomElement.vue,builder.config.js,vueform.config.js) with a Custom and Text element. If you add the Custom element it will print out the whole schema (via `form$.schema`).

If there's any case where this does not work, I'd appreciate if you could fork the sandbox and show it.
